### PR TITLE
Week1 Project Work

### PR DESCRIPTION
**Pull request by Harshit-Taiger**
Still need to try function score and some UI things.
But from the dev tools I am able to match counts etc.

### CODE REVIEW

File: week1/search.py [hunk 1]
```diff
@@ -91,10 +102,41 @@ def create_query(user_query, filters, sort="_score", sortDir="desc"):
     query_obj = {
         'size': 10,
         "query": {
-            "match_all": {} # Replace me with a query that both searches and filters
+            "bool": {
```
Comment (cuongthamhl) [inline week1/search.py:105]: Are you able to get good results on search query "iPad 2"?

You can try incorporating some of the popularity signals such as _**salesRankLongTerm**_. Example, https://github.com/cuongthamhl/search_with_machine_learning_course/pull/1/files#diff-3e7b3e71ea4a3e834e69c721f3a182d21f5c1b7e11639063fc2f1b5a78795d83R202

Comment (Harshit-Taiger) [reply week1/search.py:105]: Yes I still need to do this. Thanks

Comment (AnilKumarN) [reply week1/search.py:105]: Good work..Keep it going

Comment (AnilKumarN) [reply week1/search.py:105]: @Harshit-Taiger I think all these changes should be pushed only to our forks and not to **gsingers:main**

Comment (Harshit-Taiger) [reply week1/search.py:105]: Yes Anil, I forgot to create a fork, will do it properly for week 2 :)

**Pull request closed without merging**